addons/account_edi/models/account_edi_document.py

```diff
@@ -49,11 +49,13 @@ def _compute_edi_content(self):
                 move = doc.move_id
                 config_errors = doc.edi_format_id._check_move_configuration(move)
                 if config_errors:
-                    res = base64.b64encode('\n'.join(config_errors).encode('UTF-8'))
+                    res = base64.b64encode('\n'.join(config_errors).encode())
                 elif move.is_invoice(include_receipts=True) and doc.edi_format_id._is_required_for_invoice(move):
-                    res = base64.b64encode(doc.edi_format_id._get_invoice_edi_content(doc.move_id))
+                    content = doc.edi_format_id._get_invoice_edi_content(doc.move_id) or b''
```

Comment:
Are there particular conditions when `_get_invoice_edi_content` returns None?
If the method is not implemented in an edi localisation - `b''` is already the default - maybe we should consider fixing the relevant `_get_invoice_edi_content` instead - as this would make the download feature useless.

Reply:
Maybe we fix l10n_mx_edi...
https://github.com/odoo/enterprise/pull/22758